cslabs-backend/Models/UserModule.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using CSLabsBackend.Util;
+using Microsoft.EntityFrameworkCore;
+
+namespace CSLabsBackend.Models
+{
+    public class UserModule
+    {
+        public int id { get; set; }
+        public int  UserId { get; set; }
+        public int  ModuleId { get; set; }
+        public DateTime TerminateDate { get; set; }
```

Comment:
This field should be named `TerminationDate` not `TerminateDate `. When you renamed this or any field, be sure to generate a migration.